lib/cc/engine/csslint.rb

```diff
@@ -20,7 +20,7 @@ def run
               next unless node.name == "error"
 
               lint = node.attributes
-              check_name = lint["source"].value
+              check_name = lint["identifier"].value
```

Comment:
Nope. Any check-prefixing is meant to be done from the outside. RuboCop's current behavior, notwithstanding, that's the future we want to build for -- so says @brynary anyway.